workflows/argo/scream-prognostic-run.yaml

```diff
@@ -0,0 +1,104 @@
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: scream-prognostic-run
+spec:
+  entrypoint: driver
+  volumes:
+  - name: gcp-credentials-user-gcp-sa
+    secret:
+      secretName: gcp-key
+  - name: data-volume
+    persistentVolumeClaim:
+      claimName: scream-storage-claim
+  - name: dshm
+    emptyDir:
+      medium: Memory
+  templates:
+  - name: driver
+    inputs:
+      parameters:
+      - {name: config, value: "/tmp/180.yaml"}
+      - {name: precompiled_case, value: "true"}
+      - {name: memory, value: "700Gi"}
+      - {name: cpu, value: "180"}
+    steps:
+      - - name: choose-node-pool
+          template: choose-node-pool
+          arguments:
+            parameters:
+            - {name: cpu-request, value: "{{inputs.parameters.cpu}}"}
+            - {name: cpu-cutoff, value: "16"}
```

Comment:
The docs say for the c2-standard-30 machine type, vCPU=30, can the cutoff be set higher or are there other considerations that make the default choice 16?

Reply:
No real reason, I picked 16 because that was the number of cores I could test on my vm. Changing to 30 to match the number of cpu c2-standard has.